server/conf/application.conf

```diff
@@ -246,20 +246,24 @@ addressIndex {
       //      hybridIndex = "hybrid_latest_no_history"
       //      hybridIndex = "hybrid_39_100316_1542302693911"
       //      hybridIndex = "hybrid_39_100316_1544631791803"
-      hybridIndex = "index_full_nohist"
```

Comment:
This conf file is getting messy. It should only have the default values. Branch specific ones should be in the dev conf file.